packages/react-dev-utils/detectPort.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
This should have the original license, not FB's.

Unfortunately this makes our situation more complicated as I'd like to avoid mixing licenses in one package.

Ideally we should either use original package, or use a fork from either of our personal repos (so that we don't bring a different license here).

While we're looking at it, can we check what Zeit's `serve` uses? Maybe we can just use that (since I think it worked with `0.0.0.0` fine).